src/content_cao.h

```diff
@@ -141,9 +141,10 @@ class GenericCAO : public ClientActiveObject
 	virtual bool getSelectionBox(aabb3f *toset) const;
 
 	v3f getPosition();
-	inline float getYaw() const
+
+	inline const v3f & getRotation()
 	{
-		return m_yaw;
+	    return m_rotation;
```

Comment:
`inline const v3f &getRotation()`
and please use tabs to indent

Reply:
Fixed all requested. Also reviewed all code again for whitespaces and spaces that were used for indents.